models/leafclause.js

```diff
@@ -77,8 +77,8 @@ module.exports = Clause.extend({
   },
   initialize: function(attrs, options) {
     // initialize value manually (since it is polymorphic)
-    if (attrs) {
-      this.value = this._initializeValue(attrs.value, options);
+    if (this.value !== undefined) {
+      this.value = this._initializeValue(this.value, options);
```

Comment:
This is the source of incompatibility with 4.7.0+? Definitely a subtle detail. Also interesting that null worked previously.